libs/components/src/lib/alert/alert.template.ts

```diff
@@ -60,13 +65,13 @@ export const AlertTemplate = (context: VividElementDefinitionContext) => {
 
 	return html<Alert>`
 	<${elevationTag} class="elevation" dp='8' exportparts="vvd-theme-alternate">
-		<div class="${getControlClasses}">
+		<div class="${getControlClasses}" ${ref('_controlEl')}>
 			<div
 				part="vvd-theme-alternate"
 				class="${getClasses}"
 				role="alert"
 				aria-hidden="${(x) => (x.open ? 'false' : 'true')}"
-				${(x) => (!x.open ? 'hidden' : '')}
+				hidden="${(x) => (!x.open ? '' : null)}"
```

Comment:
Nit, but boolean attrs should be set like this:
```suggestion
				?hidden="${(x) => !x.open}"
```